src/domain/open311Backend/api/utils.ts

```diff
@@ -0,0 +1,45 @@
+import { Dirty, Clean } from "../types";
+import Message from "./Message";
+
+type PipeFunction<T> = (arg: T) => T;
+export const pipe = <T>(...fns: PipeFunction<T>[]) => (x: T) =>
+  fns.reduce((v, f) => f(v), x);
+
+export function removeNull(obj: Dirty): Clean {
+  const filteredObject = {};
+
+  Object.entries(obj).forEach(([key, value]) => {
+    if (value !== null) {
+      filteredObject[key] = value;
+    }
+  });
+
+  return filteredObject as Clean;
+}
+
+export function toSnakeCase(obj: Clean): Clean {
```

Comment:
There is a lodash function for this (lazy people like me often use those.. :-) )
And this implementation is also fine!